doc/source/whatsnew/v0.20.3.txt

```diff
@@ -55,7 +54,8 @@ Indexing
 I/O
 ^^^
 
--- Bug in ``pd.read_csv()`` in which files weren't opened as binary files by the C engine on Windows, causing EOF characters mid-field, which would fail (:issue:`16039`, :issue:`16559`, :issue`16675`)
+- Bug in ``pd.read_csv()`` in which files weren't opened as binary files by the C engine on Windows, causing EOF characters mid-field, which would fail (:issue:`16039`, :issue:`16559`, :issue`16675`)
+- Bug in ``.to_latex()`` where ``bold_rows`` was wrongly specified to be True by default, whereas in reality row labels remained non-bold whatever parameter provided. (:issue:`16707`)
```

Comment:
Maybe just say here that the default behavior is the same (not bold), but can now be overridden correctly.